miro_preprocessor/Makefile

```diff
@@ -0,0 +1,53 @@
+ROOT = $(shell git rev-parse --show-toplevel)
+
+ifneq ($(ROOT), $(shell pwd))
+	include $(ROOT)/shared.Makefile
+endif
+
+
+xml_to_json_converter-build: xml_to_json_converter-test $(ROOT)/.docker/image_builder
+	$(ROOT)/scripts/run_docker_in_docker.sh \
+		-v $(ROOT):/repo \
+		image_builder:latest \
+		--project=xml_to_json_converter \
+		--file=/repo/miro_preprocessor/xml_to_json_converter.Dockerfile
+
+xml_to_json_converter-test: $(ROOT)/.docker/python3.6_ci
+	docker run \
+		-v $(ROOT)/miro_preprocessor/xml_to_json_converter:/data \
+		-e OP=test -e FIND_MATCH_PATHS='/data' \
+		python3.6_ci:latest
+
+xml_to_json_converter-publish: xml_to_json_converter-build $(ROOT)/.docker/publish_service_to_aws
+	$(ROOT)/scripts/run_docker_with_aws_credentials.sh \
+	    -v /var/run/docker.sock:/var/run/docker.sock \
+		-v $(ROOT):/repo \
+		publish_service_to_aws:latest \
+		--project=xml_to_json_converter \
+		--infra-bucket=platform-infra
+
+miro_preprocessor-test: xml_to_json_converter-test
+
+miro_preprocessor-build: xml_to_json_converter-build
+
+miro_preprocessor-publish: xml_to_json_converter-publish
+
+miro_preprocessor-terraform-plan: $(ROOT)/.docker/terraform_ci
+	$(ROOT)/scripts/run_docker_with_aws_credentials.sh \
+		-v $(ROOT)/miro_preprocessor:/data \
+		-v $(ROOT)/terraform:/terraform \
+		-e OP=plan \
+		terraform_ci:latest
```

Comment:
While we’re thinking about changing the helper scripts, sorting out the `terraform` stuff might be useful as well.